frontend/src/RegisterByDate.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./RegisterByDate.css";
+import Arrow from "./Arrow.svg";
+import { render } from "@testing-library/react";
+
+function goBack() {
+  <link to="/.App.js"></link>;
+}
+function registerToDate() {}
```

Comment:
registerToDate and registerByDate are very similar names and could be confusing, maybe one of them can be just register/registration or something else? but its up to you, you can keep the names as is